### PR TITLE
Implemented new items up to September 17, 2024

### PR DESCRIPTION
**Pull request by Atemo**
* **Addressed Issue(s)**: -

* **Server Mode**: renewal

* **Description of Pull Request**: 

Also added new maps in mapcache

### CODE REVIEW

File: src/map/script_constants.hpp [hunk 1]
```diff
@@ -9186,6 +9215,11 @@
 	export_constant_npc(JT_4_EP21_HOWELL_S);
 	export_constant_npc(JT_4_EP21_TAN_S);
 	export_constant_npc(JT_4_M_VACATION_MARAM);
+	export_constant_npc(JT_4_CLB_SS_FC);
+	export_constant_npc(JT_4_CLB_SS_TZ);
```
Comment (AoShinRO) [inline src/map/script_constants.hpp:9219]:
```suggestion
```

File: src/map/script_constants.hpp [hunk 2]
```diff
@@ -9186,6 +9215,11 @@
...
+	export_constant_npc(JT_4_CLB_SS_AJ);
```
Comment (AoShinRO) [inline src/map/script_constants.hpp:9220]:
```suggestion
```

File: src/map/script_constants.hpp [hunk 3]
```diff
@@ -9186,6 +9215,11 @@
...
+	export_constant_npc(JT_4_CLB_SS_LK);
```
Comment (AoShinRO) [inline src/map/script_constants.hpp:9221]:
```suggestion
```

**Review by AoShinRO — COMMENTED** [review]
we have already added these

File: src/map/script_constants.hpp [hunk 4]
```diff
@@ -9186,6 +9215,11 @@
...
+	export_constant_npc(JT_4_SMART_ANDRE);
```
[the remark this answers is not in the document]
Comment (AoShinRO) [reply src/map/script_constants.hpp:9222]: sorry idk how to suggest changes in more than 1 line in git haha

[the remark this answers is not in the document]
Comment (Atemo) [reply src/map/script_constants.hpp:9222]: No worry the PR just need to be updated against master

File: db/re/item_db_usable.yml [hunk 5]
```diff
@@ -78855,6 +78902,387 @@ Body:
...
+      NoDrop: true
+      NoTrade: true
+      NoSell: true
+      NoCart: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      getgroupitem(IG_2024SPORTS_BOX);
+  - Id: 103307
```
Comment (Pokye) [inline db/re/item_db_usable.yml:79069]: Doesn't this item need a function in the script to activate the character slot?

Comment (Atemo) [reply db/re/item_db_usable.yml:79069]: Maybe, I will leave that to you. Thanks!

**Pull request merged**